app/room.py

```diff
@@ -0,0 +1,116 @@
+import random
+from os import sys, path
+
+from app.fellow import Fellow
+from app.staff import Staff
+from app.person import Person
+
+sys.path.append(path.dirname(path.dirname(
+    path.dirname(path.abspath(__file__)))))
+
+
+class Room():
+
+    total_number_of_rooms = 0
+    room_list = {}
+
+    def __init__(self, room_name):
+        self.name = room_name.capitalize()
+        self.is_full = False
+        self.room_members = []
+        Room.total_number_of_rooms += 1
+
+    def get_room_name(self):
+        return self.name
+
+    def set_room_name(self, new_room_name):
+        if not new_room_name:
+            return "Room name can't be empty"
+        self.name = new_room_name.capitalize()
+
+    def add_room_members(self, person):
+        if not isinstance(person, Person):
+            print("Only a person, staff, or fellow instance can be added")
+            return False
+        self.room_members.append(person)
+        if len(self.room_members) == self.maximum_capacity:
+            self.is_full = True
+        return True
+
+    def get_room_members(self):
+        return self.room_members
+
+    def remove_member(self, person):
+        if not isinstance(person, Person):
+            print("Only a person, staff, or fellow instance can be removed")
+            return False
+        self.room_members.remove(person)
+
+    @classmethod
+    def add_room(cls, room):
+        if not isinstance(room, Room):
+            print("Only a room, office, or livingspace instance can be added")
+            return False
+        cls.room_list[room.name] = room
```

Comment:
hmmm...this does not look professional..Having a map with the key being one of the values of an instance while the value being the entire instance itself..

This does not look any different from mapping something to itself..

Maybe `room_list` can just be a list of room instances instead..